158 text problems in tutorial text

Heldereim opened the pull request:
Changed the text and some images of the tutorial screens to decrease bloat and make things slightly more clear.

Heldereim committed:
Updated tutorial screen text

Heldereim committed:
Changed picture for the storage tutorial tab

linuskar reviewed (changes requested):
- Update storage tab image, as the increment and decrement of 5 was changed to 10.
![image](https://github.com/user-attachments/assets/4422e02b-b02b-4dac-836b-f2379dadae50)

Feel like its kinda unnecessary to mention that research entries can be free since there are none. Also maybe update research entry image for this.
![image](https://github.com/user-attachments/assets/38c7c0e7-c3d4-49c6-883d-4fec42646206)

Heldereim committed:
Merge branch 'main' into 158-text-problems-in-tutorial-text

Heldereim committed:
Updates for merge

Changed image in tutorial for storage.
Changed text in research tutorial to account for there being no free research entries.

linuskar reviewed (changes requested):
Update of research entry image

linuskar reviewed (changes requested):
Also there are errors: 
![image](https://github.com/user-attachments/assets/73260cc9-2428-4a12-b9c3-f6dd05ae777a)

Heldereim committed:
Changed research entry tutorial image

Heldereim committed:
Removed signals that were unneeded

linuskar reviewed (approved):
LGTM
